src/MissionManager/SurveyComplexItem.cc

```diff
@@ -1087,7 +1093,7 @@ void SurveyComplexItem::_rebuildTransectsPhase1WorkerSinglePolygon(bool refly)
     if (_loadedMissionItemsParent) {
         _loadedMissionItems.clear();
         _loadedMissionItemsParent->deleteLater();
-        _loadedMissionItemsParent = NULL;
+        _loadedMissionItemsParent = nullptr;
```

Comment:
I've been using the Q_NULLPTR thing instead of NULL. I think we should come to some concensus on that. I'm hoping Q_NULLPTR would be more future proof than a direct C++ thing?

Reply:
I agree. At some point I will look into a global search and replace...